src/tools/blast/config/BlastFormData.ts

```diff
@@ -58,7 +58,7 @@ export default Object.freeze({
     selected: '',
   }),
   [BlastFields.targetDb]: Object.freeze({
-    fieldName: 'database',
+    fieldName: 'targetDb',
```

Comment:
This should probably have had been changed the other way around, as 'database' is what is used in the internal job parameters (`FormParameters`) and on the server (`ServerParameters`)

Reply:
Done